mne/io/bti/bti.py

```diff
@@ -894,7 +894,8 @@ def _read_bti_header_pdf(pdf_fname):
 def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
     """ Read bti PDF header
     """
-    info = _read_bti_header_pdf(pdf_fname)
+
+    info = _read_bti_header_pdf(pdf_fname) if pdf_fname is not None else dict()
```

Comment:
at some point, we should try to standardize these internals. @Eric89GXL implemented `_raw_extras` to deal with system-specific info. it would also make it easier for enhancements down the line